app/db.js

```diff
@@ -1,6 +1,17 @@
 'use strict';
 
+const env = process.env.NODE_ENV || 'development';
 const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/fadpack');
 
-module.exports = mongoose;
+const name = `fadpack-${env}`;
```

Comment:
We're using es6 template strings. https://developers.google.com/web/updates/2015/01/ES6-Template-Strings